src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -87,39 +90,58 @@ public class DependenciesFinder
   public Set<ResolvedDependency> findResolvedDependencies(
       Project project,
       boolean allConfigurations,
-      Map<String, String> variantAttributes)
+      Map<String, String> variantAttributes,
+      boolean excludeCompileOnlyDependencies)
```

Comment:
Personal preference, but is it worth overloading this method with `excludeCompileOnlyDependencies` param? This method can call the overloaded method with a default value `false`. Might save you adding `false` to all the calls in `DependenciesFinderTest`

Reply:
> Personal preference, but is it worth overloading this method with `excludeCompileOnlyDependencies` param? This method can call the overloaded method with a default value `false`. Might save you adding `false` to all the calls in `DependenciesFinderTest`

`findResolvedDependencies()` is a public method and it's called for the OssIndexAudit task where `excludeCompileOnlyDependencies` is passed down:
https://github.com/sonatype-nexus-community/scan-gradle-plugin/blob/b0d82525834e292462cee412d91a5437ed4753dc/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/OssIndexAuditTask.java#L81

Fot that reason alone I would prefer to keep it in a single method.
But as the list of params here becomes larger, I'd be inclined to create a new class holding all this "configuration" values.